Kitodo/src/main/java/org/kitodo/production/services/data/MassImportService.java

```diff
@@ -99,10 +107,17 @@ public List<CsvRecord> parseLines(List<String> lines, String separator) throws I
                      .withSkipLines(0)
                      .withCSVParser(parser)
                      .build()) {
+            int targetNumberOfColumns = -1;
             for (String[] entries : csvReader.readAll()) {
                 if (isSingleEmptyEntry(entries)) {
                     continue; // Skip processing this line
                 }
+                // throw exception different number of metadata entries were parsed for different CSV lines
+                if (targetNumberOfColumns >= 0 && entries.length != targetNumberOfColumns) {
+                    throw new KitodoCsvImportException(Helper.getTranslation("massImport.separatorCountMismatchEntries",
+                            Pattern.quote(separator)));
+                }
+                targetNumberOfColumns = entries.length;
```

Comment:
Why is the targetNumberOfColumns determined for every row? Is it not enough to calculate it once - for the first row and then compatre every following row to it?